ios/RNTrackPlayer/RNTrackPlayer.swift

```diff
@@ -131,6 +131,10 @@ public class RNTrackPlayer: RCTEventEmitter, AudioPlayerDelegate {
     @objc(setupPlayer:resolver:rejecter:)
     public func setupPlayer(config: [String: Any], resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) {
         do {
+            let autoWait: Bool = config["waitForBuffer"] as? Bool ?? false
+            if autoWait {
+                player.automaticallyWaitsToMinimizeStalling = true
+            }
```

Comment:
I think you could either drop `?? false` or remove the `if` statement and set `autoWait` every time.

Reply:
iirc, swift needs the default value because the option being unwrapped could be nil. But yeah, if it is always evaluating to a boolean I could set the option here every time.